serverless.yml

```diff
@@ -9,7 +9,7 @@ custom:
   stackName: ${self:service}-${self:provider.stage}
   xray: 'Active'  # AWS Xray tracing of requests, queries
   db:
-    dbname: "TEMPLATE"
+    dbname: "TEMPLATEx"
```

Comment:
is this a typo?

Reply:
yes, I have to change the name to deploy it to AWS because TEMPLATE is a reserved name in pg...